### DISCUSSION TITLE
mixcr analysis for 2 (or more) different experiments

### DISCUSSION THREAD

**Discussion by jyeoneeee** [milaboratory/mixcr#1334] [Q&A] [answered]
Hi, I am using mixcr for TCR analysis from bulk-seq fastq samples.
Here is my question,
is there are any way to use 2 (or more) different experiment condition samples to compare TCR seq with mixcr?

For example,
I ran bulk-seq about 3-months ago and prepare to do another sequencing run for different conditions.
In this case, is it possible to assemble all the samples together for comparing TCR clones with mixcr?
or by using other TCR analyzing tools such as Immunarch, or is it impossible?

Thanks and wait for your kind answer,

Jiyeon

**Comment by mizraelson** [comment, marked as answer]
Hi, 
Yes, there are many ways you can compare TCR repertoires across multiple samples using MiXCR. What features do you want to investigate: shared clonotypes between  the samples, V\J usage, Diversity, CDR3 region biophysical characteristics etc.

If you can share the details of what you metrics you wanna use, I can help you with the commands.

Also, you can read about it [here](https://mixcr.com/mixcr/reference/mixcr-postanalysis/).

**Comment by jeremymsimon** [comment]
Hi @mizraelson, I'm interested in this as well. I have 10x scTCR data I processed with `mixcr analyze 10x-sc-xcr-vdj` using v4.5.0 with the IMGT reference. I have ~20 samples in total, and want to ultimately want to find clonotypes expanded in certain samples relative to others, and plot their relative abundance across samples. I've attempted some of this using `immunoarch`, with AIRR-formatted tables I exported with `mixcr exportAirr`, however `immunarch` seems to only support paired-chain analysis with cellranger-formatted files. I would want to match my cells based on CDR3 amino acid + V + J for TRA+TRB

Alternatively if there's some way of exporting MiXCR's tables into a 10x-like format, there may be a way to run the clonotype tracking in `immunarch`, as mentioned [here](https://github.com/immunomind/immunarch/issues/222).

Any help on this would be appreciated!
